Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3639,7 +3646,7 @@ def add_modified_remote_offenses(client: Client,
         new_context_data.update({MIRRORED_OFFENSES_QUERIED_CTX_KEY: mirrored_offenses_queries})
         new_context_data.update({MIRRORED_OFFENSES_FINISHED_CTX_KEY: finished_offenses_queue})
 
-    new_context_data.update({LAST_MIRROR_KEY: current_last_update})
+    new_context_data.update({LAST_MIRROR_KEY: new_last_update_modified, LAST_MIRROR_CLOSED_KEY: new_last_update_closed})
```

Comment:
didn't we say we don't need additional key in context for closed incidents?

Reply:
There is another edge case:
1. We query for modified and closed offenses.
2. The latest modified time is 3. The latest closed time is 7.
3. We will save 7 as the last update time.
4. In the next mirroring run, we can miss offenses that were modified between 4-6.

The safer option is just to store two separate timestamps.